src/SmartTransactionsController.ts

```diff
@@ -613,8 +749,13 @@ export default class SmartTransactionsController extends BaseController<
       smartTransactionsState: {
         ...this.state.smartTransactionsState,
         liveness,
```

Comment:
`liveness` should only be set if the networkClientId's chainId matches the globally selected chainId

Reply:
yep good catch!

Reply:
done here: https://github.com/MetaMask/smart-transactions-controller/pull/237/commits/caf5ac0a59b73823ef6977cb480bf4c3ec2b5b22